build/raise-ivy-projects-version/raise.sh

```diff
@@ -30,12 +30,7 @@ downloadEngine(){
     zipped=$(find "${workDir}" -maxdepth 1 -name "*.zip")
     unzip -qq "${zipped}" -d "${workDir}/engine"
     rm "${zipped}"
-  fi
-  jar="*thirdparty*.jar"
-  if [ -z "$(ls ${workDir}/engine/dropins/${jar})" ]; then
-    curl --output ${workDir}/engine/p2.zip https://drone.ivyteam.io/unzip/maven/prefixed/ivy-core/ch.ivyteam.ivy/ch.ivyteam.ivy.core.p2/${ivyVersion}-SNAPSHOT
```

Comment:
process migrations will fail, if we can't load the bpm.exec.thirdparty extensions ... so yes, we still need this code.